src/ape/managers/chain.py

```diff
@@ -408,18 +414,82 @@ def outgoing(self) -> Iterator[ReceiptAPI]:
         """
         All outgoing transactions, from earliest to latest.
         """
-        explorer = self.provider.network.explorer
-        explorer_receipts = (
-            [r for r in explorer.get_account_transactions(self.address)] if explorer else []
-        )
-        for receipt in explorer_receipts:
-            if receipt.sender != self.address:
-                # Likely ``self.address`` is a contract.
-                # Cache the receipts by their sender instead and skip them here.
-                self.chain_manager.history.append(receipt)
-                continue
+
+        start_nonce = 0
+        stop_nonce = self.__len__() - 1  # just to cache this value
```

Comment:
Should this be `len(self) - 1` just for syntactic consistency with the method below this one?